app/javascript/angular/code/services/_mobile_sessions.js

```diff
@@ -24,10 +24,15 @@ export const mobileSessions = (
     this.scope.params = params;
   };
 
-  let prevMapPosition = {
-    bounds: map.getBounds(),
-    zoom: map.getZoom()
-  };
+  let prevMapPosition = {};
+  if (params.get("selectedSessionIds").length === 1) {
+    prevMapPosition = params.get("prevMapPosition");
+  } else {
+    prevMapPosition = {
+      bounds: map.getBounds(),
+      zoom: map.getZoom()
+    };
+  }
```

Comment:
this is the same as in `fixed_sessions`, right? what do you think of extracting it?